app/src/main/java/com/netmera/netmerafintech/ui/home/ManageCardActivity.kt

```diff
@@ -1,31 +1,64 @@
 package com.netmera.netmerafintech.ui.home
 
+import android.app.Activity
+import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.netmera.netmerafintech.R
+import com.netmera.netmerafintech.data.CardType
+import com.netmera.netmerafintech.data.model.Card
 import com.netmera.netmerafintech.databinding.ActivityManageCardBinding
+import com.netmera.netmerafintech.utils.parcelable
 import com.netmera.netmerafintech.utils.toast
 
 class ManageCardActivity: AppCompatActivity() {
+
+    companion object {
+        private const val ARG_CARD = "ARG_CARD"
+
+        @JvmStatic
```

Comment:
We don't need `JvmStatic` in this project since all classes written with Kotlin :) You can remove them from the entire project.